src/rgl.cpp

```diff
@@ -1080,12 +1080,24 @@ void rglDisplayFramebuffers()
 
 void rglUpdate()
 {
-  int i;
-  
-  if (old_vi_origin == vi_origin) {
```

Comment:
Currently processRDPlist calls rglUpdate
If you remove this rglUpdate will be called hundreds of times per frame and everything will slow down a lot

I'm looking for a better place to call rglUpdate, maybe fullsync is a good place